src/Nethermind/Nethermind.Network/P2P/Subprotocols/Eth/V62/Messages/BlockBodiesMessageSerializer.cs

```diff
@@ -102,14 +102,15 @@ private int GetWithdrawalsLength(Withdrawal[] withdrawals)
                 // quite significant allocations (>0.5%) here based on a sample 3M blocks sync
                 // (just on these delegates)
                 Transaction[] transactions = ctx.DecodeArray(_txDecoder);
+                Transaction[] inclusionListTransactions = ctx.DecodeArray(_txDecoder);
```

Comment:
When we add new field to block structure we have to add it in the end. And we have to check if inclusion list is encoded. Check how withdrawals are processed lower